doc/changes/changes_0.5.0.md

```diff
@@ -0,0 +1,11 @@
+# dynamodb-virtual-schema 0.5.0, released 2020-08-XX
+ 
+## Summary
+
+
```

Comment:
Add summary. One sentence with user-friendly description is enough in this case.